package.json

```diff
@@ -1,156 +1,16 @@
-    "@rollup/plugin-node-resolve": "^13.0.6",
-    "@rollup/plugin-replace": "^3.0.0",
-    "@semantic-release/changelog": "^6.0.0",
-    "@semantic-release/git": "^10.0.0",
-    "@wdio/cli": "^7.3.1",
-    "@wdio/jasmine-framework": "^7.4.6",
-    "@wdio/junit-reporter": "^7.4.2",
-    "@wdio/local-runner": "^7.4.6",
-    "@wdio/selenium-standalone-service": "^7.5.2",
-    "@wdio/spec-reporter": "^7.4.3",
-    "@wdio/static-server-service": "^7.5.7",
-    "@webex/test-users": "^1.157.0",
-    "babel-eslint": "^10.0.3",
-    "babel-loader": "^8.0.6",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "buffer": "^6.0.3",
-    "clean-webpack-plugin": "^3.0.0",
-    "crypto-browserify": "^3.12.0",
-    "css-loader": "^3.4.2",
-    "dotenv": "^8.6.0",
-    "eslint": "^6.8.0",
-    "eslint-config-airbnb": "^18.0.1",
-    "eslint-config-prettier": "^6.10.0",
-    "eslint-config-react": "^1.1.7",
-    "eslint-plugin-import": "^2.20.1",
-    "eslint-plugin-jest": "^23.6.0",
-    "eslint-plugin-jsdoc": "^21.0.0",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-mocha": "^6.2.2",
-    "eslint-plugin-prettier": "^3.1.2",
-    "eslint-plugin-react": "^7.18.3",
-    "eslint-plugin-react-hooks": "^1.7.0",
-    "file-loader": "^5.1.0",
-    "html-loader": "^0.5.5",
-    "html-webpack-plugin": "^5.6.0",
-    "husky": "^4.2.3",
-    "node-sass": "^9.0.0",
-    "os-browserify": "^0.3.0",
-    "postcss": "^8.2.13",
-    "postcss-url": "^10.1.3",
-    "prettier": "^1.19.1",
-    "prop-types": "^15.7.2",
-    "querystring-es3": "^0.2.1",
-    "react": "18.3.1",
-    "react-dom": "18.3.1",
-    "rollup": "^2.59.0",
-    "rollup-plugin-postcss": "^4.0.0",
-    "sass-loader": "^10.5.2",
-    "semantic-release": "^18.0.0",
-    "stream-browserify": "^3.0.0",
-    "style-loader": "^1.1.3",
-    "url": "^0.11.3",
-    "util": "^0.12.5",
-    "vm-browserify": "^1.1.2",
-    "webpack": "^5.93.0",
-    "webpack-cli": "^5.1.4",
-    "webpack-dev-server": "^5.0.4"
-  },
-  "peerDependencies": {
-    "prop-types": "^15.7.2",
-    "react": "18.3.1",
-    "react-dom": "18.3.1",
-    "webex": "2.60.2"
-  },
-  "babel": {
-    "presets": [
-      "@babel/preset-env",
-      "@babel/preset-react"
-    ],
-    "plugins": [
-      "@babel/plugin-proposal-export-default-from",
-      "@babel/plugin-proposal-nullish-coalescing-operator",
-      "@babel/plugin-proposal-object-rest-spread",
-      "@babel/plugin-proposal-optional-chaining",
-      "@babel/plugin-transform-regenerator",
-      "@babel/plugin-transform-runtime"
-    ]
-  },
-  "commitlint": {
-    "extends": [
-      "@commitlint/config-conventional"
-    ],
-    "rules": {
-      "scope-case": [
-        2,
-        "always",
-        [
-          "lower-case",
-          "pascal-case"
-        ]
-      ]
-    }
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run test:eslint",
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
-    }
+    "node-gyp": "^10.2.0"
```

Comment:
This dev dependency is required to build the `deasync` dependency from `sdk-component-adapter`